rubin_sim/scheduler/model_observatory/model_observatory.py

```diff
@@ -280,6 +280,7 @@ def return_conditions(self):
             self.seeing_fwhm_eff[key].fill(np.nan)
         # Use the model to get the seeing at this time and airmasses.
         fwhm_500 = self.seeing_data(current_time)
+        self.conditions.fwhm_500 = fwhm_500
```

Comment:
I'm ok with changing it like that. I do think it's possible that the "fwhm 500" would be useful to have in the conditions object, as it would be the seeing from the DIMM, which we might want to record in the conditions object (which Tiago is also saving to disk with each snapshot). Otherwise we have to try to reconstruct it from either the seeing over the sky we receive (which would mean going backwards through the seeing model) or try to pull the "right" DIMM measurement out of the EFD (I don't know how often the DIMM measurement is recorded, but it's likely much more often than we get in the conditions for the FBS).